apps/wallet/src/components/CheckedInput.tsx

```diff
@@ -1,31 +1,33 @@
 import type { FC } from 'react';
-import { useState } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import type { InputProps } from '@walless/gui';
 import { Input } from '@walless/gui';
 
 type Props = {
-	checkFunction: (value?: string) => string | undefined;
+	errorText?: string;
+	onFocusOut?: (value?: string) => void;
```

Comment:
Also, I think it's not good to bind value param to `onFocusOut`, as I mention below, keeping some state inside this component might be better